source/guide_focalboard.rst

```diff
@@ -0,0 +1,98 @@
+.. highlight:: console
+
+.. author:: Thomas Kammerer <https://kammerer.tk/>
+
+.. tag:: web
+.. tag:: project-management
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/focalboard.svg
+      :align: center
+
+##########
+Focalboard
+##########
+
+.. tag_list::
+
+Focalboard_ is an open source, self-hosted alternative to Trello, Notion, and Asana. 
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+
+Prerequisites
+=============
+
+Your website domain needs to be set up:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Download the latest release_ to your home directory and extract the application.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ wget https://github.com/mattermost/focalboard/releases/download/v0.6.1/focalboard-linux.tar.gz
+ [isabell@stardust ~]$ tar xzfv focalboard-linux.tar.gz
+ [isabell@stardust ~]$
+
+
+Configuration
+=============
+
+You need to change ``"localOnly": true`` to ``"localOnly": false`` in ``~/focalboard-app/config.json``
+
```

Comment:
Complementing my comment from above, the database configuration also needs to be adjusted if PostgreSQL is used.

```
If PostgreSQL is used, the database configuration also needs to be adjusted. Edit the database config in the file ``~/focalboard-app/config.json``. These two values must be changed:

 .. warning:: Replace ``<db_user>``, ``<db_password>`` and ``<db_name>`` with your values.

 .. code-block:: json
   "dbtype": "postgres",
   "dbconfig": "postgres://<db_user>:<db_password>@localhost/<db_name>?sslmode=disable&connect_timeout=10",
 For example:

 .. code-block:: json
   "dbtype": "postgres",
   "dbconfig": "postgres://isabell_focalboard:sup3r-s3cr3t@localhost/focalboard?sslmode=disable&connect_timeout=10",
```

Reply:
:+1: